wallet/how-to/register-token.md

```diff
@@ -46,8 +66,89 @@ try {
 }
 ```
 
+:::note
+If the chain ID of your token doesn't match the user's network, they can get unexpected results.
+We recommend [detecting the user's network chain ID](../get-started/detect-network.md) and
+[prompting them to switch chains](../reference/rpc-api.md#wallet_switchethereumchain), if necessary.
```

Comment:
this should link to the API Playground method: `../reference/wallet_switchethereumchain/`